USAGE.md

```diff
@@ -0,0 +1,142 @@
+Usage
+=====
+
+## Requirements
+
+You need at least Node `v4.0.0` and npm `2`.
+
+## Create your project
+Install the required tools: yo, gulp, bower
+
+```sh
+$ npm install -g yo jspm typings bower
+```
+
+Install `generator-fountain-webapp`
+
+```sh
+$ npm install -g generator-fountain-webapp
+```
+Make a new directory, and cd into it:
+
+```sh
+mkdir my-new-project && cd $_
+``` 
+
+Run yo fountain-webapp:
+
+```sh
+yo fountain-webapp
+```
+
+## Yo options
+`yo fountain-generator --help` or `yo fountain-generator -h` for help. All options are not required. If not provided, default values will be used.
+
+* `--skip-cache` Do not remember prompt answers, default is `false`
+* `--skip-install` Do not automatically install dependencies, default is `false`
+* `--framework` Description for framework
+* `--modules` Description for modules manager
+* `--css` Description for css preprocessor
+* `--js` Description for js preprocessor
+
+## Use Gulp tasks
+
+- gulp or gulp build to build an optimized version of your application in /dist
+- gulp serve to launch a browser sync server on your source files
+- gulp serve:dist to launch a server on your optimized application
+- gulp test to launch your unit tests with Karma
+- gulp test:auto to launch your unit tests with Karma in watch mode
+
+## Directory structure
+
+Every component of the app get its own directory which contains the component code (depending on the framework, it can be `*.(js|ts)` or/and `*.component.(js|ts)` or/and `*.controller.(js|ts)` or/and `*.service.(js|ts)` or/and `*.directive.(js|ts)` and `*.spec.(js|ts)`.
```

Comment:
That's obsolete talking about directives ^^ Even for ng1 we removed `.component.js` files suffixes